2018-07-03 14:06:42.073 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 14:06:42.075 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 14:07:42.075 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 14:07:42.077 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 14:08:03.706 25931 DEBUG oslo_concurrency.lockutils [req-9728c614-b31b-4f1a-b2fe-ded387e470db 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Acquired semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:08:04.552 25931 DEBUG neutron.agent.dhcp.agent [req-2063c650-4037-43d0-902a-d052a848c8af - - - - -] Calling driver for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-07-03 14:08:04.553 25931 DEBUG neutron.agent.linux.utils [req-2063c650-4037-43d0-902a-d052a848c8af - - - - -] Unable to access /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:08:04.554 25931 DEBUG neutron.agent.linux.dhcp [req-2063c650-4037-43d0-902a-d052a848c8af - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 on network 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1267
2018-07-03 14:08:04.555 25931 DEBUG neutron.agent.linux.dhcp [req-2063c650-4037-43d0-902a-d052a848c8af - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 on network 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1288
2018-07-03 14:08:08.520 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:08.546 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:08.562 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:08.578 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:08.592 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:08.607 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', 'link', 'set', 'tapa158160e-58', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:08.633 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-o', 'link', 'show', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:08.653 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-07-03 14:08:08.962 25931 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tapa158160e-58) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-07-03 14:08:08.963 25931 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-07-03 14:08:08.964 25931 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tapa158160e-58) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-07-03 14:08:08.965 25931 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'a158160e-582f-44ff-9ce3-51b211b7587e', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:61:6a:5f'})), record=tapa158160e-58) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-07-03 14:08:09.016 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tapa158160e-58', 'address', 'fa:16:3e:61:6a:5f'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.036 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.061 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tapa158160e-58', 'netns', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.094 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', 'link', 'set', 'tapa158160e-58', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.107 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', 'link', 'set', 'tapa158160e-58', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.128 25931 DEBUG oslo_concurrency.lockutils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Acquired semaphore "iptables-qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:08:09.131 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.158 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.192 25931 DEBUG neutron.agent.linux.iptables_manager [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-07-03 14:08:09.193 25931 DEBUG oslo_concurrency.lockutils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Releasing semaphore "iptables-qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:08:09.194 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-o', 'link', 'show', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.210 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', 'addr', 'show', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.224 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-4', 'addr', 'add', '11.77.1.2/24', 'scope', 'global', 'dev', 'tapa158160e-58', 'brd', '11.77.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.241 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-4', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.252 25931 DEBUG neutron.agent.linux.dhcp [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Setting IPv4 gateway for dhcp netns on net 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 to 11.77.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1164
2018-07-03 14:08:09.253 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.77.1.1', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.273 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-6', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.288 25931 DEBUG neutron.agent.linux.dhcp [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Cleaning stale devices for network 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1369
2018-07-03 14:08:09.289 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.300 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.316 25931 DEBUG neutron.agent.linux.dhcp [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Building initial lease file: /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:623
2018-07-03 14:08:09.318 25931 DEBUG neutron.agent.linux.dhcp [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Done building initial lease file /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/leases with contents:
1530706089 fa:16:3e:61:6a:5f 11.77.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:651
2018-07-03 14:08:09.319 25931 DEBUG neutron.agent.linux.dhcp [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Building host file: /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-07-03 14:08:09.320 25931 DEBUG neutron.agent.linux.dhcp [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Done building host file /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-07-03 14:08:09.322 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Unable to access /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:08:09.323 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host', '--addn-hosts=/var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tapa158160e-58', '--dhcp-range=set:tag0,11.77.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.369 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:08:09.372 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:08:09.373 25931 DEBUG neutron.agent.linux.external_process [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] No process started for 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-07-03 14:08:09.376 25931 DEBUG oslo_concurrency.lockutils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Releasing semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:08:09.377 25931 DEBUG oslo_concurrency.lockutils [req-9728c614-b31b-4f1a-b2fe-ded387e470db 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Acquired semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:08:09.954 25931 DEBUG neutron.agent.dhcp.agent [req-a1570f3e-2f3b-4b70-8a3a-52ea22de887c - - - - -] Calling driver for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-07-03 14:08:09.955 25931 DEBUG neutron.agent.linux.dhcp [req-a1570f3e-2f3b-4b70-8a3a-52ea22de887c - - - - -] Building host file: /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-07-03 14:08:09.957 25931 DEBUG neutron.agent.linux.dhcp [req-a1570f3e-2f3b-4b70-8a3a-52ea22de887c - - - - -] Done building host file /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-07-03 14:08:09.962 25931 DEBUG neutron.agent.linux.utils [req-a1570f3e-2f3b-4b70-8a3a-52ea22de887c - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '30385'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.981 25931 DEBUG neutron.agent.linux.dhcp [req-a1570f3e-2f3b-4b70-8a3a-52ea22de887c - - - - -] Reloading allocations for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-07-03 14:08:09.983 25931 DEBUG neutron.agent.linux.utils [req-a1570f3e-2f3b-4b70-8a3a-52ea22de887c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-4', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:09.997 25931 DEBUG neutron.agent.linux.utils [req-a1570f3e-2f3b-4b70-8a3a-52ea22de887c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-6', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:10.014 25931 DEBUG neutron.agent.linux.utils [req-a1570f3e-2f3b-4b70-8a3a-52ea22de887c - - - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:08:10.015 25931 DEBUG neutron.agent.linux.utils [req-a1570f3e-2f3b-4b70-8a3a-52ea22de887c - - - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:08:10.016 25931 DEBUG neutron.agent.linux.external_process [req-a1570f3e-2f3b-4b70-8a3a-52ea22de887c - - - - -] No process started for 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-07-03 14:08:10.017 25931 DEBUG oslo_concurrency.lockutils [req-a1570f3e-2f3b-4b70-8a3a-52ea22de887c - - - - -] Releasing semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:08:10.018 25931 DEBUG oslo_concurrency.lockutils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Acquired semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:08:10.019 25931 DEBUG neutron.agent.dhcp.agent [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-07-03T12:08:05Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6a004e1f-1012-46ae-a043-867313a91668', u'ip_address': u'11.77.1.2'}], id=a158160e-582f-44ff-9ce3-51b211b7587e, mac_address=fa:16:3e:61:6a:5f, name=, network_id=626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1, port_security_enabled=False, project_id=0f18d1d8963a4acead108afba45f713d, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=0f18d1d8963a4acead108afba45f713d, updated_at=2018-07-03T12:08:06Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:409
2018-07-03 14:08:10.020 25931 DEBUG oslo_concurrency.lockutils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Releasing semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:08:10.021 25931 DEBUG oslo_concurrency.lockutils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Acquired semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:08:10.021 25931 INFO neutron.agent.dhcp.agent [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-07-03T12:08:05Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6a004e1f-1012-46ae-a043-867313a91668', u'ip_address': u'11.77.1.2'}], id=a158160e-582f-44ff-9ce3-51b211b7587e, mac_address=fa:16:3e:61:6a:5f, name=, network_id=626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1, port_security_enabled=False, project_id=0f18d1d8963a4acead108afba45f713d, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=0f18d1d8963a4acead108afba45f713d, updated_at=2018-07-03T12:08:08Z
2018-07-03 14:08:10.023 25931 DEBUG neutron.agent.dhcp.agent [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Calling driver for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-07-03 14:08:10.024 25931 DEBUG neutron.agent.linux.dhcp [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Building host file: /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-07-03 14:08:10.026 25931 DEBUG neutron.agent.linux.dhcp [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Done building host file /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-07-03 14:08:10.036 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '30385'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:10.061 25931 DEBUG neutron.agent.linux.dhcp [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Reloading allocations for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-07-03 14:08:10.063 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-4', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:10.078 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-6', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:10.092 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:08:10.093 25931 DEBUG neutron.agent.linux.utils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:08:10.094 25931 DEBUG neutron.agent.linux.external_process [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] No process started for 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-07-03 14:08:10.096 25931 DEBUG oslo_concurrency.lockutils [req-f403e266-73c8-439a-9f5d-69e1da4a497c - - - - -] Releasing semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:08:40.281 25931 DEBUG oslo_concurrency.lockutils [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Acquired semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:08:40.282 25931 INFO neutron.agent.dhcp.agent [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-07-03T12:08:39Z, description=, device_id=92e91ba1-e24f-4b0c-bf0f-001a7dd050a5, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6a004e1f-1012-46ae-a043-867313a91668', u'ip_address': u'11.77.1.1'}], id=01bf919c-8011-4399-83fa-821d90f98083, mac_address=fa:16:3e:c6:47:a3, name=, network_id=626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1, port_security_enabled=False, project_id=0f18d1d8963a4acead108afba45f713d, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=0f18d1d8963a4acead108afba45f713d, updated_at=2018-07-03T12:08:39Z
2018-07-03 14:08:40.283 25931 DEBUG neutron.agent.dhcp.agent [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Calling driver for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-07-03 14:08:40.285 25931 DEBUG neutron.agent.linux.dhcp [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Building host file: /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-07-03 14:08:40.290 25931 DEBUG neutron.agent.linux.dhcp [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Done building host file /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-07-03 14:08:40.295 25931 DEBUG neutron.agent.linux.utils [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Running command (rootwrap daemon): ['kill', '-HUP', '30385'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:40.311 25931 DEBUG neutron.agent.linux.dhcp [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Reloading allocations for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-07-03 14:08:40.313 25931 DEBUG neutron.agent.linux.utils [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-4', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:40.326 25931 DEBUG neutron.agent.linux.utils [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-6', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:40.337 25931 DEBUG neutron.agent.linux.utils [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:08:40.338 25931 DEBUG neutron.agent.linux.utils [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:08:40.339 25931 DEBUG neutron.agent.linux.external_process [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] No process started for 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-07-03 14:08:40.340 25931 DEBUG oslo_concurrency.lockutils [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Releasing semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:08:41.668 25931 DEBUG oslo_concurrency.lockutils [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Acquired semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:08:41.669 25931 INFO neutron.agent.dhcp.agent [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-07-03T12:08:39Z, description=, device_id=92e91ba1-e24f-4b0c-bf0f-001a7dd050a5, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6a004e1f-1012-46ae-a043-867313a91668', u'ip_address': u'11.77.1.1'}], id=01bf919c-8011-4399-83fa-821d90f98083, mac_address=fa:16:3e:c6:47:a3, name=, network_id=626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1, port_security_enabled=False, project_id=0f18d1d8963a4acead108afba45f713d, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=0f18d1d8963a4acead108afba45f713d, updated_at=2018-07-03T12:08:39Z
2018-07-03 14:08:41.670 25931 DEBUG neutron.agent.dhcp.agent [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Calling driver for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-07-03 14:08:41.674 25931 DEBUG neutron.agent.linux.dhcp [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Building host file: /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-07-03 14:08:41.676 25931 DEBUG neutron.agent.linux.dhcp [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Done building host file /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-07-03 14:08:41.680 25931 DEBUG neutron.agent.linux.utils [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Running command (rootwrap daemon): ['kill', '-HUP', '30385'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:41.692 25931 DEBUG neutron.agent.linux.dhcp [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Reloading allocations for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-07-03 14:08:41.693 25931 DEBUG neutron.agent.linux.utils [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-4', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:41.705 25931 DEBUG neutron.agent.linux.utils [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-6', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:41.717 25931 DEBUG neutron.agent.linux.utils [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:08:41.718 25931 DEBUG neutron.agent.linux.utils [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:08:41.718 25931 DEBUG neutron.agent.linux.external_process [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] No process started for 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-07-03 14:08:41.719 25931 DEBUG oslo_concurrency.lockutils [req-87e8ec44-b2e2-45ba-bb23-acc0c1b214bb 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Releasing semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:08:42.075 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 14:08:42.077 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 14:08:44.668 25931 DEBUG oslo_concurrency.lockutils [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Acquired semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:08:44.669 25931 INFO neutron.agent.dhcp.agent [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-07-03T12:08:39Z, description=, device_id=92e91ba1-e24f-4b0c-bf0f-001a7dd050a5, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6a004e1f-1012-46ae-a043-867313a91668', u'ip_address': u'11.77.1.1'}], id=01bf919c-8011-4399-83fa-821d90f98083, mac_address=fa:16:3e:c6:47:a3, name=, network_id=626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1, port_security_enabled=False, project_id=0f18d1d8963a4acead108afba45f713d, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=0f18d1d8963a4acead108afba45f713d, updated_at=2018-07-03T12:08:40Z
2018-07-03 14:08:44.670 25931 DEBUG neutron.agent.dhcp.agent [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Calling driver for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-07-03 14:08:44.672 25931 DEBUG neutron.agent.linux.dhcp [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Building host file: /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-07-03 14:08:44.674 25931 DEBUG neutron.agent.linux.dhcp [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Done building host file /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-07-03 14:08:44.678 25931 DEBUG neutron.agent.linux.utils [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '30385'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:44.697 25931 DEBUG neutron.agent.linux.dhcp [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Reloading allocations for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-07-03 14:08:44.698 25931 DEBUG neutron.agent.linux.utils [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-4', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:44.709 25931 DEBUG neutron.agent.linux.utils [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-6', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:44.721 25931 DEBUG neutron.agent.linux.utils [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:08:44.722 25931 DEBUG neutron.agent.linux.utils [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:08:44.722 25931 DEBUG neutron.agent.linux.external_process [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] No process started for 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-07-03 14:08:44.723 25931 DEBUG oslo_concurrency.lockutils [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Releasing semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:08:45.270 25931 DEBUG oslo_concurrency.lockutils [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Acquired semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:08:45.271 25931 INFO neutron.agent.dhcp.agent [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-07-03T12:08:39Z, description=, device_id=92e91ba1-e24f-4b0c-bf0f-001a7dd050a5, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6a004e1f-1012-46ae-a043-867313a91668', u'ip_address': u'11.77.1.1'}], id=01bf919c-8011-4399-83fa-821d90f98083, mac_address=fa:16:3e:c6:47:a3, name=, network_id=626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1, port_security_enabled=False, project_id=0f18d1d8963a4acead108afba45f713d, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=0f18d1d8963a4acead108afba45f713d, updated_at=2018-07-03T12:08:44Z
2018-07-03 14:08:45.272 25931 DEBUG neutron.agent.dhcp.agent [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Calling driver for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-07-03 14:08:45.274 25931 DEBUG neutron.agent.linux.dhcp [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Building host file: /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-07-03 14:08:45.276 25931 DEBUG neutron.agent.linux.dhcp [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Done building host file /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-07-03 14:08:45.280 25931 DEBUG neutron.agent.linux.utils [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '30385'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:45.291 25931 DEBUG neutron.agent.linux.dhcp [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Reloading allocations for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-07-03 14:08:45.293 25931 DEBUG neutron.agent.linux.utils [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-4', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:45.304 25931 DEBUG neutron.agent.linux.utils [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-6', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:08:45.315 25931 DEBUG neutron.agent.linux.utils [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:08:45.316 25931 DEBUG neutron.agent.linux.utils [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:08:45.317 25931 DEBUG neutron.agent.linux.external_process [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] No process started for 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-07-03 14:08:45.317 25931 DEBUG oslo_concurrency.lockutils [req-28e4224d-2d52-4d76-8eaf-e4624a1201d1 - - - - -] Releasing semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:08:53.393 25931 DEBUG oslo_concurrency.lockutils [req-37bfcbf7-21ce-4402-925e-9f19b61ea60f 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Acquired semaphore "dhcp-agent-network-lock-2ce34dd7-df32-4a77-980f-3c1215fa4847" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:08:53.394 25931 DEBUG oslo_concurrency.lockutils [req-37bfcbf7-21ce-4402-925e-9f19b61ea60f 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Releasing semaphore "dhcp-agent-network-lock-2ce34dd7-df32-4a77-980f-3c1215fa4847" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:08:56.615 25931 DEBUG oslo_concurrency.lockutils [req-f980a68a-474e-4d1b-87a1-5a5f1e4d7427 - - - - -] Acquired semaphore "dhcp-agent-network-lock-2ce34dd7-df32-4a77-980f-3c1215fa4847" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:08:56.616 25931 DEBUG oslo_concurrency.lockutils [req-f980a68a-474e-4d1b-87a1-5a5f1e4d7427 - - - - -] Releasing semaphore "dhcp-agent-network-lock-2ce34dd7-df32-4a77-980f-3c1215fa4847" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:08:57.380 25931 DEBUG oslo_concurrency.lockutils [req-f980a68a-474e-4d1b-87a1-5a5f1e4d7427 - - - - -] Acquired semaphore "dhcp-agent-network-lock-2ce34dd7-df32-4a77-980f-3c1215fa4847" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:08:57.381 25931 DEBUG oslo_concurrency.lockutils [req-f980a68a-474e-4d1b-87a1-5a5f1e4d7427 - - - - -] Releasing semaphore "dhcp-agent-network-lock-2ce34dd7-df32-4a77-980f-3c1215fa4847" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:09:30.934 25931 DEBUG oslo_concurrency.lockutils [req-fa4c19ec-fe96-4662-93ca-f172d3f1dcde 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Acquired semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:09:30.935 25931 INFO neutron.agent.dhcp.agent [req-fa4c19ec-fe96-4662-93ca-f172d3f1dcde 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-07-03T12:09:28Z, description=, device_id=8e66ad3e-886f-40a4-b83f-588b8045a04c, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6a004e1f-1012-46ae-a043-867313a91668', u'ip_address': u'11.77.1.3'}], id=219706e6-a5ea-4d90-b506-625ebc0beab9, mac_address=fa:16:3e:34:eb:d9, name=, network_id=626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1, port_security_enabled=True, project_id=0f18d1d8963a4acead108afba45f713d, revision_number=3, security_groups=[u'e7c5d191-d536-4317-ad11-dae00065df01'], status=DOWN, tags=[], tenant_id=0f18d1d8963a4acead108afba45f713d, updated_at=2018-07-03T12:09:28Z
2018-07-03 14:09:30.936 25931 DEBUG neutron.agent.dhcp.agent [req-fa4c19ec-fe96-4662-93ca-f172d3f1dcde 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Calling driver for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-07-03 14:09:30.938 25931 DEBUG neutron.agent.linux.dhcp [req-fa4c19ec-fe96-4662-93ca-f172d3f1dcde 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Building host file: /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-07-03 14:09:30.945 25931 DEBUG neutron.agent.linux.dhcp [req-fa4c19ec-fe96-4662-93ca-f172d3f1dcde 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Done building host file /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-07-03 14:09:30.959 25931 DEBUG neutron.agent.linux.utils [req-fa4c19ec-fe96-4662-93ca-f172d3f1dcde 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Running command (rootwrap daemon): ['kill', '-HUP', '30385'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:09:31.022 25931 DEBUG neutron.agent.linux.dhcp [req-fa4c19ec-fe96-4662-93ca-f172d3f1dcde 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Reloading allocations for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-07-03 14:09:31.023 25931 DEBUG neutron.agent.linux.utils [req-fa4c19ec-fe96-4662-93ca-f172d3f1dcde 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-4', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:09:31.049 25931 DEBUG neutron.agent.linux.utils [req-fa4c19ec-fe96-4662-93ca-f172d3f1dcde 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-6', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:09:31.066 25931 DEBUG neutron.agent.linux.utils [req-fa4c19ec-fe96-4662-93ca-f172d3f1dcde 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:09:31.067 25931 DEBUG neutron.agent.linux.utils [req-fa4c19ec-fe96-4662-93ca-f172d3f1dcde 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:09:31.068 25931 DEBUG neutron.agent.linux.external_process [req-fa4c19ec-fe96-4662-93ca-f172d3f1dcde 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] No process started for 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-07-03 14:09:31.070 25931 DEBUG oslo_concurrency.lockutils [req-fa4c19ec-fe96-4662-93ca-f172d3f1dcde 05d003873bb549e19a3a19b31f84eb02 0f18d1d8963a4acead108afba45f713d - - -] Releasing semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:09:33.141 25931 DEBUG oslo_concurrency.lockutils [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Acquired semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:09:33.142 25931 INFO neutron.agent.dhcp.agent [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-07-03T12:09:28Z, description=, device_id=8e66ad3e-886f-40a4-b83f-588b8045a04c, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6a004e1f-1012-46ae-a043-867313a91668', u'ip_address': u'11.77.1.3'}], id=219706e6-a5ea-4d90-b506-625ebc0beab9, mac_address=fa:16:3e:34:eb:d9, name=, network_id=626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1, port_security_enabled=True, project_id=0f18d1d8963a4acead108afba45f713d, revision_number=4, security_groups=[u'e7c5d191-d536-4317-ad11-dae00065df01'], status=DOWN, tags=[], tenant_id=0f18d1d8963a4acead108afba45f713d, updated_at=2018-07-03T12:09:31Z
2018-07-03 14:09:33.144 25931 DEBUG neutron.agent.dhcp.agent [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Calling driver for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-07-03 14:09:33.149 25931 DEBUG neutron.agent.linux.dhcp [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Building host file: /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-07-03 14:09:33.153 25931 DEBUG neutron.agent.linux.dhcp [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Done building host file /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-07-03 14:09:33.161 25931 DEBUG neutron.agent.linux.utils [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '30385'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:09:33.189 25931 DEBUG neutron.agent.linux.dhcp [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Reloading allocations for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-07-03 14:09:33.190 25931 DEBUG neutron.agent.linux.utils [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-4', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:09:33.211 25931 DEBUG neutron.agent.linux.utils [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-6', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:09:33.236 25931 DEBUG neutron.agent.linux.utils [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:09:33.237 25931 DEBUG neutron.agent.linux.utils [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:09:33.238 25931 DEBUG neutron.agent.linux.external_process [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] No process started for 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-07-03 14:09:33.238 25931 DEBUG oslo_concurrency.lockutils [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Releasing semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:09:34.199 25931 DEBUG oslo_concurrency.lockutils [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Acquired semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:09:34.201 25931 INFO neutron.agent.dhcp.agent [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-07-03T12:09:28Z, description=, device_id=8e66ad3e-886f-40a4-b83f-588b8045a04c, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'6a004e1f-1012-46ae-a043-867313a91668', u'ip_address': u'11.77.1.3'}], id=219706e6-a5ea-4d90-b506-625ebc0beab9, mac_address=fa:16:3e:34:eb:d9, name=, network_id=626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1, port_security_enabled=True, project_id=0f18d1d8963a4acead108afba45f713d, revision_number=5, security_groups=[u'e7c5d191-d536-4317-ad11-dae00065df01'], status=DOWN, tags=[], tenant_id=0f18d1d8963a4acead108afba45f713d, updated_at=2018-07-03T12:09:33Z
2018-07-03 14:09:34.203 25931 DEBUG neutron.agent.dhcp.agent [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Calling driver for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-07-03 14:09:34.204 25931 DEBUG neutron.agent.linux.dhcp [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Building host file: /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-07-03 14:09:34.206 25931 DEBUG neutron.agent.linux.dhcp [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Done building host file /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-07-03 14:09:34.212 25931 DEBUG neutron.agent.linux.utils [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '30385'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:09:34.244 25931 DEBUG neutron.agent.linux.dhcp [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Reloading allocations for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-07-03 14:09:34.245 25931 DEBUG neutron.agent.linux.utils [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-4', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:09:34.272 25931 DEBUG neutron.agent.linux.utils [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-6', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:09:34.300 25931 DEBUG neutron.agent.linux.utils [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:09:34.301 25931 DEBUG neutron.agent.linux.utils [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:09:34.301 25931 DEBUG neutron.agent.linux.external_process [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] No process started for 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-07-03 14:09:34.311 25931 DEBUG oslo_concurrency.lockutils [req-0ad816a5-6179-4c0b-b10e-152ecaaf885b 8d51c6595f7b4b648f3775454a985b0c e3731f8202904eebb3e6658e91a6b0b5 - - -] Releasing semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:09:42.075 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 14:09:42.081 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.006s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 14:10:06.580 25931 DEBUG oslo_concurrency.lockutils [req-4a4a29da-ba68-4012-8ea4-83da033a558e 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Acquired semaphore "dhcp-agent-network-lock-2ce34dd7-df32-4a77-980f-3c1215fa4847" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:10:06.581 25931 DEBUG oslo_concurrency.lockutils [req-4a4a29da-ba68-4012-8ea4-83da033a558e 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Releasing semaphore "dhcp-agent-network-lock-2ce34dd7-df32-4a77-980f-3c1215fa4847" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:10:08.507 25931 DEBUG oslo_concurrency.lockutils [req-4a4a29da-ba68-4012-8ea4-83da033a558e 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Acquired semaphore "dhcp-agent-network-lock-2ce34dd7-df32-4a77-980f-3c1215fa4847" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:10:08.509 25931 DEBUG oslo_concurrency.lockutils [req-4a4a29da-ba68-4012-8ea4-83da033a558e 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Releasing semaphore "dhcp-agent-network-lock-2ce34dd7-df32-4a77-980f-3c1215fa4847" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:10:42.076 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 14:10:42.078 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 14:11:42.076 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 14:11:42.079 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 14:12:42.078 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 14:12:42.081 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 14:13:42.079 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 14:13:42.082 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 14:14:42.079 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 14:14:42.082 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 14:14:53.557 25931 DEBUG oslo_concurrency.lockutils [req-fe1b74fc-ef5d-49ca-bdd3-e0c95743e716 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Acquired semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:14:53.558 25931 DEBUG neutron.agent.dhcp.agent [req-fe1b74fc-ef5d-49ca-bdd3-e0c95743e716 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Calling driver for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-07-03 14:14:53.560 25931 DEBUG neutron.agent.linux.utils [req-fe1b74fc-ef5d-49ca-bdd3-e0c95743e716 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'dhcp_release', 'tapa158160e-58', '11.77.1.3', 'fa:16:3e:34:eb:d9'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:14:53.589 25931 DEBUG neutron.agent.linux.dhcp [req-fe1b74fc-ef5d-49ca-bdd3-e0c95743e716 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Building host file: /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-07-03 14:14:53.596 25931 DEBUG neutron.agent.linux.dhcp [req-fe1b74fc-ef5d-49ca-bdd3-e0c95743e716 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Done building host file /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-07-03 14:14:53.615 25931 DEBUG neutron.agent.linux.utils [req-fe1b74fc-ef5d-49ca-bdd3-e0c95743e716 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Running command (rootwrap daemon): ['kill', '-HUP', '30385'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:14:53.637 25931 DEBUG neutron.agent.linux.dhcp [req-fe1b74fc-ef5d-49ca-bdd3-e0c95743e716 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Reloading allocations for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-07-03 14:14:53.639 25931 DEBUG neutron.agent.linux.utils [req-fe1b74fc-ef5d-49ca-bdd3-e0c95743e716 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-4', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:14:53.654 25931 DEBUG neutron.agent.linux.utils [req-fe1b74fc-ef5d-49ca-bdd3-e0c95743e716 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-6', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:14:53.679 25931 DEBUG neutron.agent.linux.utils [req-fe1b74fc-ef5d-49ca-bdd3-e0c95743e716 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:14:53.681 25931 DEBUG neutron.agent.linux.utils [req-fe1b74fc-ef5d-49ca-bdd3-e0c95743e716 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:14:53.681 25931 DEBUG neutron.agent.linux.external_process [req-fe1b74fc-ef5d-49ca-bdd3-e0c95743e716 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] No process started for 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-07-03 14:14:53.682 25931 DEBUG oslo_concurrency.lockutils [req-fe1b74fc-ef5d-49ca-bdd3-e0c95743e716 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Releasing semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:15:42.081 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 14:15:42.084 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 14:15:48.600 25931 DEBUG oslo_concurrency.lockutils [req-c2c2eaf1-7019-4ba5-b695-95563f44d5ac 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Acquired semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:15:48.601 25931 DEBUG neutron.agent.dhcp.agent [req-c2c2eaf1-7019-4ba5-b695-95563f44d5ac 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Calling driver for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-07-03 14:15:48.603 25931 DEBUG neutron.agent.linux.utils [req-c2c2eaf1-7019-4ba5-b695-95563f44d5ac 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'dhcp_release', 'tapa158160e-58', '11.77.1.1', 'fa:16:3e:c6:47:a3'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:15:48.622 25931 DEBUG neutron.agent.linux.dhcp [req-c2c2eaf1-7019-4ba5-b695-95563f44d5ac 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Building host file: /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-07-03 14:15:48.626 25931 DEBUG neutron.agent.linux.dhcp [req-c2c2eaf1-7019-4ba5-b695-95563f44d5ac 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Done building host file /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-07-03 14:15:48.631 25931 DEBUG neutron.agent.linux.utils [req-c2c2eaf1-7019-4ba5-b695-95563f44d5ac 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Running command (rootwrap daemon): ['kill', '-HUP', '30385'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:15:48.644 25931 DEBUG neutron.agent.linux.dhcp [req-c2c2eaf1-7019-4ba5-b695-95563f44d5ac 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Reloading allocations for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-07-03 14:15:48.645 25931 DEBUG neutron.agent.linux.utils [req-c2c2eaf1-7019-4ba5-b695-95563f44d5ac 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-4', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:15:48.659 25931 DEBUG neutron.agent.linux.utils [req-c2c2eaf1-7019-4ba5-b695-95563f44d5ac 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1', 'ip', '-6', 'route', 'list', 'dev', 'tapa158160e-58'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:15:48.677 25931 DEBUG neutron.agent.linux.utils [req-c2c2eaf1-7019-4ba5-b695-95563f44d5ac 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:15:48.679 25931 DEBUG neutron.agent.linux.utils [req-c2c2eaf1-7019-4ba5-b695-95563f44d5ac 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:15:48.680 25931 DEBUG neutron.agent.linux.external_process [req-c2c2eaf1-7019-4ba5-b695-95563f44d5ac 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] No process started for 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-07-03 14:15:48.681 25931 DEBUG oslo_concurrency.lockutils [req-c2c2eaf1-7019-4ba5-b695-95563f44d5ac 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Releasing semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:16:15.949 25931 DEBUG oslo_concurrency.lockutils [req-8185c573-bba2-4bf3-98ce-ce2660423e09 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Acquired semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:16:15.950 25931 DEBUG neutron.agent.dhcp.agent [req-8185c573-bba2-4bf3-98ce-ce2660423e09 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Calling driver for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-07-03 14:16:15.954 25931 DEBUG neutron.agent.linux.utils [req-8185c573-bba2-4bf3-98ce-ce2660423e09 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Running command (rootwrap daemon): ['kill', '-9', '30385'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:16:15.970 25931 DEBUG neutron.agent.linux.utils [req-8185c573-bba2-4bf3-98ce-ce2660423e09 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-07-03 14:16:16.004 25931 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tapa158160e-58) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-07-03 14:16:16.103 25931 DEBUG neutron.agent.linux.utils [req-4a665bd5-cd41-4b5c-978b-1c9f40b3c89b - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:16:16.117 25931 DEBUG neutron.agent.linux.utils [req-4a665bd5-cd41-4b5c-978b-1c9f40b3c89b - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:16:16.134 25931 DEBUG oslo_concurrency.lockutils [req-4a665bd5-cd41-4b5c-978b-1c9f40b3c89b - - - - -] Releasing semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:16:26.173 25931 DEBUG oslo_concurrency.lockutils [req-d1f85c06-64ca-4621-a772-330a9a027f16 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Acquired semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:16:26.445 25931 DEBUG neutron.agent.linux.utils [req-3e3fc767-f050-4366-aeb9-e3e91c2634c7 - - - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:16:26.446 25931 DEBUG neutron.agent.linux.utils [req-3e3fc767-f050-4366-aeb9-e3e91c2634c7 - - - - -] Unable to access /var/lib/neutron/external/pids/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:16:26.446 25931 DEBUG neutron.agent.linux.external_process [req-3e3fc767-f050-4366-aeb9-e3e91c2634c7 - - - - -] No process started for 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-07-03 14:16:26.447 25931 DEBUG neutron.agent.dhcp.agent [req-3e3fc767-f050-4366-aeb9-e3e91c2634c7 - - - - -] Calling driver for network: 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-07-03 14:16:26.448 25931 DEBUG neutron.agent.linux.utils [req-3e3fc767-f050-4366-aeb9-e3e91c2634c7 - - - - -] Unable to access /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:16:26.448 25931 DEBUG neutron.agent.linux.utils [req-3e3fc767-f050-4366-aeb9-e3e91c2634c7 - - - - -] Unable to access /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-07-03 14:16:26.449 25931 DEBUG neutron.agent.linux.external_process [req-3e3fc767-f050-4366-aeb9-e3e91c2634c7 - - - - -] No process started for 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-07-03 14:16:26.449 25931 DEBUG neutron.agent.linux.dhcp [req-3e3fc767-f050-4366-aeb9-e3e91c2634c7 - - - - -] Impossibile accedere a /var/lib/neutron/dhcp/626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:268
2018-07-03 14:16:26.449 25931 DEBUG neutron.agent.linux.dhcp [req-3e3fc767-f050-4366-aeb9-e3e91c2634c7 - - - - -] No interface exists for network 626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 destroy /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1482
2018-07-03 14:16:26.482 25931 DEBUG neutron.agent.linux.utils [req-b8dbe9f5-5149-4fd2-8050-2b6c3b386374 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-07-03 14:16:26.505 25931 DEBUG neutron.agent.linux.dhcp [req-b8dbe9f5-5149-4fd2-8050-2b6c3b386374 - - - - -] Namespace already deleted: qdhcp-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1 _destroy_namespace_and_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:246
2018-07-03 14:16:26.507 25931 DEBUG oslo_concurrency.lockutils [req-b8dbe9f5-5149-4fd2-8050-2b6c3b386374 - - - - -] Releasing semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:16:27.543 25931 DEBUG oslo_concurrency.lockutils [req-d1f85c06-64ca-4621-a772-330a9a027f16 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Acquired semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-07-03 14:16:27.546 25931 DEBUG oslo_concurrency.lockutils [req-d1f85c06-64ca-4621-a772-330a9a027f16 41f81b51431f44bd96d368d9b49cd4bd 688c18a1c85f4751b329806136ced2ee - - -] Releasing semaphore "dhcp-agent-network-lock-626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-07-03 14:16:35.058 25931 DEBUG neutron.agent.dhcp.agent [req-9820d73c-a569-48ca-9b70-fe6401e171bd - - - - -] resync (626f3d8a-e6f4-4fcd-8fc4-041bbaa4a8f1): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:255
2018-07-03 14:16:35.059 25931 INFO neutron.agent.dhcp.agent [req-9820d73c-a569-48ca-9b70-fe6401e171bd - - - - -] Synchronizing state
2018-07-03 14:16:35.572 25931 INFO neutron.agent.dhcp.agent [req-3641336f-ccc4-44ee-8178-36084bce918f - - - - -] All active networks have been fetched through RPC.
2018-07-03 14:16:35.573 25931 INFO neutron.agent.dhcp.agent [-] Starting network 2ce34dd7-df32-4a77-980f-3c1215fa4847 dhcp configuration
2018-07-03 14:16:35.574 25931 INFO neutron.agent.dhcp.agent [-] Finished network 2ce34dd7-df32-4a77-980f-3c1215fa4847 dhcp configuration
2018-07-03 14:16:35.574 25931 INFO neutron.agent.dhcp.agent [req-3641336f-ccc4-44ee-8178-36084bce918f - - - - -] Synchronizing state complete
2018-07-03 14:16:42.081 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 14:16:42.084 25931 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
